<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Recommended_standard</class>
 <widget class="QMainWindow" name="Recommended_standard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="Init">
            <property name="styleSheet">
             <string notr="true">font: 13pt &quot;Ubuntu&quot;;</string>
            </property>
            <property name="text">
             <string>Init</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_rs485">
            <property name="styleSheet">
             <string notr="true">font: 13pt &quot;Ubuntu&quot;;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QPushButton" name="write">
              <property name="styleSheet">
               <string notr="true">font: 13pt &quot;Ubuntu&quot;;</string>
              </property>
              <property name="text">
               <string>Write</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="read">
              <property name="styleSheet">
               <string notr="true">font: 13pt &quot;Ubuntu&quot;;</string>
              </property>
              <property name="text">
               <string>Read</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTextEdit" name="textEdit_rs485">
            <property name="styleSheet">
             <string notr="true">font: 13pt &quot;Ubuntu&quot;;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="3" column="0">
         <widget class="QPushButton" name="s_pushButton">
          <property name="text">
           <string>s</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="m_pushButton">
          <property name="text">
           <string>m</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="b_pushButton">
          <property name="text">
           <string>b</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QPushButton" name="e_pushButton">
          <property name="text">
           <string>e</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="d_pushButton">
          <property name="text">
           <string>d</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="c_pushButton">
          <property name="text">
           <string>c</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="g_pushButton">
          <property name="text">
           <string>g</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QPushButton" name="pushButton_5">
          <property name="text">
           <string>5</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="y_pushButton">
          <property name="text">
           <string>y</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="a_pushButton">
          <property name="text">
           <string>a</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QPushButton" name="f_pushButton">
          <property name="text">
           <string>f</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="h_pushButton">
          <property name="text">
           <string>h</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="i_pushButton">
          <property name="text">
           <string>i</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="j_pushButton">
          <property name="text">
           <string>j</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QPushButton" name="k_pushButton">
          <property name="text">
           <string>k</string>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="QPushButton" name="l_pushButton">
          <property name="text">
           <string>l</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="n_pushButton">
          <property name="text">
           <string>n</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="o_pushButton">
          <property name="text">
           <string>o</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QPushButton" name="p_pushButton">
          <property name="text">
           <string>p</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QPushButton" name="q_pushButton">
          <property name="text">
           <string>q</string>
          </property>
         </widget>
        </item>
        <item row="2" column="5">
         <widget class="QPushButton" name="r_pushButton">
          <property name="text">
           <string>r</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="t_pushButton">
          <property name="text">
           <string>t</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="u_pushButton">
          <property name="text">
           <string>u</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QPushButton" name="v_pushButton">
          <property name="text">
           <string>v</string>
          </property>
         </widget>
        </item>
        <item row="3" column="4">
         <widget class="QPushButton" name="w_pushButton">
          <property name="text">
           <string>w</string>
          </property>
         </widget>
        </item>
        <item row="3" column="5">
         <widget class="QPushButton" name="x_pushButton">
          <property name="text">
           <string>x</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="z_pushButton">
          <property name="text">
           <string>z</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="pushButton_1">
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>2</string>
          </property>
         </widget>
        </item>
        <item row="4" column="4">
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>3</string>
          </property>
         </widget>
        </item>
        <item row="4" column="5">
         <widget class="QPushButton" name="pushButton_4">
          <property name="text">
           <string>4</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QPushButton" name="pushButton_6">
          <property name="text">
           <string>6</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QPushButton" name="pushButton_7">
          <property name="text">
           <string>7</string>
          </property>
         </widget>
        </item>
        <item row="5" column="3">
         <widget class="QPushButton" name="pushButton_0">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="5" column="4">
         <widget class="QPushButton" name="pushButton_backspace">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="audio_resource.qrc">
            <normaloff>:/audio/thank_you_audio_file/back_space.png</normaloff>:/audio/thank_you_audio_file/back_space.png</iconset>
          </property>
         </widget>
        </item>
        <item row="5" column="5">
         <widget class="QPushButton" name="pushButton_enter">
          <property name="text">
           <string>Enter</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="close">
      <property name="styleSheet">
       <string notr="true">font: 15pt &quot;Ubuntu&quot;;</string>
      </property>
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="audio_resource.qrc"/>
 </resources>
 <connections/>
</ui>
